python/ray/tune/schedulers/pbt.py

```diff
@@ -319,6 +319,12 @@ def set_search_properties(self, metric: Optional[str],
 
     def on_trial_add(self, trial_runner: "trial_runner.TrialRunner",
                      trial: Trial):
+        if trial_runner.search_alg is not None:
```

Comment:
it can also be BasicVariantGenerator right?

Reply:
Good point, updated.